packages/simcore-sdk/src/simcore_sdk/node_data/data_manager.py

```diff
@@ -76,7 +76,8 @@ async def push(
             io_log_redirect_cb=io_log_redirect_cb,
             progress_bar=progress_bar,
         )
-    # we have a folder, so we create a compressed file
+        return

```

Comment:
thisone is very easy to miss, what about?
```
if path.is_file():
   await _push_file( ... )
else:
  async with AsyncExitStack() as stack:
    ...
```

Reply:
for now I will remove this next PR, since I'm currently chaining this